src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -2004,6 +2004,205 @@ public override void Accept(OperationVisitor visitor)
     }
 
     /// <remarks>
+    /// Represents an interpolated string expression.
+    /// </remarks>
+    internal abstract partial class BaseInterpolatedStringExpression : Operation, IInterpolatedStringExpression
```

Comment:
>internal abstract partial class BaseInterpolatedStringExpression : Operation, IInterpolatedStringExpression [](start = 4, length = 107)

Is this file really generated? I do not see changes to Operations.xml in this PR.